make.ps1

```diff
@@ -199,6 +199,53 @@ function WaitForInput
 	}
 }
 
+function ParseConfigFile($fileName)
+{
+	$reader = [System.IO.File]::OpenText($fileName)
+	while($null -ne ($line = $reader.ReadLine()))
+	{
+		if ($line.StartsWith("MOD_ID"))
```

Comment:
These checks and replaces could be pulled out into a helper function that takes in the variable name.